test/test_backpropagation.py

```diff
@@ -870,6 +870,7 @@ def test_backpropagate_multi(self):
             with self.assertRaises(ValueError):
                 backpropagate(obs, slices, operator_budget=op_budget)
 
+    # pragma: no cover
```

Comment:
This is NOT what was causing a problem here. Coverage is used to measure what lines of code in the source (not the unittests) are being executed so adding this line has no effect.
```suggestion
```

Instead, the particular CI run happened to fail due to #39. When that happens and until we improve how that unittest works, we just have to re-run the CI job.